template/template.go

```diff
@@ -108,6 +107,12 @@ func FromGenDoc(template *gendoc.Template, cfg *config.Config) (*WorkatoTemplate
 				seviceMethod := &ServiceMethod{service, method}
 
 				workatoOpt, _ := method.Option("s12.protobuf.workato.method").(*workato.MethodOptionsWorkato)
+
+				// Will exclude all the actions which have extension method with excluded flag set
+				if workatoOpt != nil && workatoOpt.Excluded {
```

Comment:
Can I suggest that we break up the nil checks?

We can get away with one `workatoOpt != nil` at the top - then `workatoOpt.Excluded`, then `Picklist`... etc.